### PR TITLE
#3324 - Income Eligibility - CSGP

### PR DESCRIPTION
**Pull request by katrina-aot-git**
Update part time assessments for CSGP.

### CODE REVIEW

File: sources/packages/web/src/components/common/students/StudentProfile.vue [hunk 1]
```diff
@@ -128,7 +128,8 @@ export default defineComponent({
   setup(props) {
     const studentDetail = ref({} as SharedStudentProfile);
     const address = ref({} as AddressAPIOutDTO);
-    const { genderDisplayFormat, sinDisplayFormat, emptyStringFiller } = useFormatters();
+    const { genderDisplayFormat, sinDisplayFormat, emptyStringFiller } =
```
Comment (guru-aot) [inline sources/packages/web/src/components/common/students/StudentProfile.vue:131]: is this change intentional?, else please revert.

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2023-2024.bpmn [hunk 2]
```diff
@@ -557,7 +557,7 @@ dmnX (comes from dmn)</bpmn:text>
     <bpmn:intermediateThrowEvent id="Event_084hvhv" name="awardEligibilityCSGP">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;  and calculatedDataPDPPDStatus = true&#10;then true&#10;else false" target="awardEligibilityCSGP" />
+          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;  and (calculatedDataPDPPDStatus = true)&#10;  and (calculatedDataTotalFamilyIncome &#60; dmnPartTimeAwardFamilySizeVariables.limitAwardCSGPThresholdIncome)&#10;then true&#10;else false&#10;" target="awardEligibilityCSGP" />
```
Comment (guru-aot) [inline sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2023-2024.bpmn:560]: remove the extra space after false

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn [hunk 3]
```diff
@@ -557,7 +557,7 @@ dmnX (comes from dmn)</bpmn:text>
     <bpmn:intermediateThrowEvent id="Event_084hvhv" name="awardEligibilityCSGP">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;  and calculatedDataPDPPDStatus = true&#10;then true&#10;else false" target="awardEligibilityCSGP" />
+          <zeebe:output source="=if (calculatedDataTotalAssessedNeed &#62;= 1)&#10;  and (calculatedDataPDPPDStatus = true)&#10;  and (calculatedDataTotalFamilyIncome &#60; dmnPartTimeAwardFamilySizeVariables.limitAwardCSGPThresholdIncome)&#10;then true&#10;else false&#10;" target="awardEligibilityCSGP" />
```
Comment (guru-aot) [inline sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn:560]: remove the extra space after false

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-decisions.dmn [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+<definitions xmlns="https://www.omg.org/spec/DMN/20191111/MODEL/" xmlns:dmndi="https://www.omg.org/spec/DMN/20191111/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:biodi="http://bpmn.io/schema/dmn/biodi/2.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" xmlns:camunda="http://camunda.org/schema/1.0/dmn" id="dmnPartTimeAssessmentDecisions" name="dmnPartTimeAssessmentDecisions" namespace="http://camunda.org/schema/1.0/dmn" exporter="Camunda Modeler" exporterVersion="5.22.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.1.0" camunda:diagramRelationId="1a575ead-1e6d-468c-901a-766a35569b38">
```
Comment (guru-aot) [inline sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-decisions.dmn:2]: please upgrade the camunda modeler and update the dmn, the version is not right

File: sources/packages/backend/workflow/test/2022-2023/parttime-assessment/eligibility/parttime-assessment-eligibility-CSGP.e2e-spec.ts [hunk 5]
```diff
@@ -7,12 +7,13 @@ import {
 describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGP.`, () => {
   it(
     "Should determine CSGP as eligible when total assessed need is greater than or equal to 1 " +
-      "and application PD/PPD status is true.",
+      "and application PD/PPD status is true and income is below the limitAwardCSGPThresholdIncome.",
```
Comment (guru-aot) [inline sources/packages/backend/workflow/test/2022-2023/parttime-assessment/eligibility/parttime-assessment-eligibility-CSGP.e2e-spec.ts:10]: can we please add one test case for family income greater than the threshold income for csgp so that checking the eligibility false., please do it for the other program year test cases too.

File: sources/packages/web/src/components/common/students/StudentProfile.vue (hunk 1 above)
Comment (katrina-aot-git) [reply sources/packages/web/src/components/common/students/StudentProfile.vue:131]: It's an old formatter update that didn't get put into another PR.

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2023-2024.bpmn (hunk 2 above)
Comment (katrina-aot-git) [reply sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2023-2024.bpmn:560]: updated

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn (hunk 3 above)
Comment (katrina-aot-git) [reply sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-2024-2025.bpmn:560]: updated

File: sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-decisions.dmn (hunk 4 above)
Comment (katrina-aot-git) [reply sources/packages/backend/workflow/src/workflow-definitions/parttime-assessment-decisions.dmn:2]: updated to 24

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @katrina-aot-git
